lib/symlink.go

```diff
@@ -32,45 +36,47 @@ func CreateSymlink(cwd string, dir string) {
 	} else {
 		err := os.Symlink(cwd, dir)
 		if err != nil {
-			logger.Fatalf(`
+			logger.Debugf(`
 		Unable to create new symlink.
 		Maybe symlink already exist. Try removing existing symlink manually.
 		Try running "unlink %s" to remove existing symlink.
 		If error persist, you may not have the permission to create a symlink at %s.
 		Error: %s
 		`, dir, dir, err)
-			os.Exit(1)
+			return errors.New(fmt.Sprintf("Unable to create new symlink %s : %s", dir, err))
```

Comment:
Why `errors.New(fmt.Sprintf(…))` rather than just `fmt.Errorf()`?
Ref: https://pkg.go.dev/errors#New

Reply:
Also please use `%q` for strings and `%v` for errors.